### ISSUE TITLE
Pointer resize: problem (Renderer Process error)

### ISSUE THREAD

**Issue by Manaberry** [jean-emmanuel/open-stage-control#308]
Using the edge of a push button to resize it with the cursor occurs the following error on the 0.32.2

```
[Renderer process error]
Uncaught ReferenceError: handleTarget is not defined
at start (src/browser/js/app/editor/index.js:235:35)
at cleanData (src/browser/js/libs/jquery.ui.js:6:0)
at _destroy (src/browser/js/libs/jquery.ui.js:7:0)
at cleanData (src/browser/js/libs/jquery.ui.js:6:0)
at _destroy (src/browser/js/libs/jquery.ui.js:7:0)
at cleanData (src/browser/js/libs/jquery.ui.js:6:0)
at cleanData (src/browser/js/libs/jquery.ui.js:6:0)
at cleanData (src/browser/js/libs/jquery.ui.js:6:0)
at cleanData (src/browser/js/libs/jquery.ui.js:6:0)
at g (node_modules/jquery/dist/jquery.slim.min.js:2:0)
```

I have to change the width directly within the field to avoid this bug.
I've tried on "push buttons" only. It occurred each time I tried.

**Comment by jean-emmanuel** [comment]
Thank you, it's fixed in git, a patch release will follow soon.

**Comment by jean-emmanuel** [comment]
Fixed in v0.32.3
